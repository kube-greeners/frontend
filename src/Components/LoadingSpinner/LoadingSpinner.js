import React from "react";
import styles from "./LoadingSpinner.module.css";

export default function LoadingSpinner() {
  return (
    <div className={styles.lds}>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  );
}