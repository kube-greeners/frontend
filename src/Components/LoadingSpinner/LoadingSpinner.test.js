import LoadingSpinner from "./LoadingSpinner";
import renderer from "react-test-renderer";

test("matches LoadingSpinner snapshot", () => {
  const tree = renderer.create(<LoadingSpinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
