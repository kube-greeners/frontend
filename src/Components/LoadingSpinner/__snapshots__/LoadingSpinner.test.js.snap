// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches LoadingSpinner snapshot 1`] = `
<div
  className="lds"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
